install/debian-installation-guide.md

```diff
@@ -32,8 +32,8 @@
    ```bash
    $ ARCH=$(arch)
    $ source /etc/os-release
-   $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_9/$(lsb_release -rs)/ /' > /etc/apt/sources.list.d/kata-containers.list"
-   $ curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_9/$(lsb_release -rs)/Release.key | sudo apt-key add -
+   $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_9/ /' > /etc/apt/sources.list.d/kata-containers.list"
```

Comment:
For reference, that change is now required I think as a result of @marcov renaming the Debian OBS repo to make it simpler: https://github.com/kata-containers/documentation/pull/306#discussion_r242983373.

Reply:
Actually the change you mentioned @jodh-intel was about renaming `/Debian_9.0/` into `/Debian_9/`. The trailing `/$(lsb_release -rs)/` directory was probably wrong since the beginning.

Proposal: what about changing it into `/Debian_$VERSION_ID/`, sourcing the variable from `/etc/os-release`?

Reply:
Oh sorry @marcov you are correct. Yep, `/Debian_${VERSION_ID}/` would wfm.